docs/data-modeling/field-types.md

```diff
@@ -45,23 +47,33 @@ Metabase recognizes the following field types:
 - **Income**
 - **Price**
 
-### Numeric
+See [Currency formatting options](../questions/sharing/visualizations/table.md#currency-formatting-options).
+
+## Numeric
+
+Metabase will treat numeric field types as [numbers](../questions/sharing/visualizations/table.md#number-formatting-options).
 
-- **Percentage**
-- **Quantity**
-- **Score**
-- **Share**. The same as percentage, so prefer "Percentage".
+- **Percentage**: Displays the number as a percentage by default.
+- **Quantity**: Displays the number as normal by default.
+- **Score**: Displays the number as normal by default.
+- **Share**: The same as percentage, so prefer "Percentage".
 
-### Profile
+See [Number formatting options](../questions/sharing/visualizations/table.md#number-formatting-options).
 
-- **Birthday**
-- **Company**
-- **Email**
-- **Owner**
-- **Subscription**
-- **User**
+## Profile
 
-### Date and Time
+Fields that deal with people.
+
+- **Birthday**: Everyone has a birthday, but birthdays only happen once a year.
```

Comment:
The note about birthdays happening once a year makes me think that if I set a field to have this type, metabase will do some kind of data validation. Is that the case?